From 514d0f1d219063f1d2816617d618b70ffff4f5d9 Mon Sep 17 00:00:00 2001
From: Soren Brinkmann <soren.brinkmann@xilinx.com>
Date: Thu, 8 Nov 2012 08:37:24 -0800
Subject: [PATCH] arm: zynq: Add dtb targets to Makefile.boot

Adding dtb targets to let the kernel build system
automatically build Zynq DT blobs.

Signed-off-by: Soren Brinkmann <soren.brinkmann@xilinx.com>
---
 arch/arm/mach-zynq/Makefile.boot |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-zynq/Makefile.boot b/arch/arm/mach-zynq/Makefile.boot
index 760a0ef..fb82c9e 100644
--- a/arch/arm/mach-zynq/Makefile.boot
+++ b/arch/arm/mach-zynq/Makefile.boot
@@ -1,3 +1,8 @@
    zreladdr-y	+= 0x00008000
 params_phys-y	:= 0x00000100
 initrd_phys-y	:= 0x00800000
+
+dtb-$(CONFIG_ARCH_ZYNQ) += zynq-zc702.dtb zynq-afx.dtb zynq-zc706.dtb \
+				zynq-zc770-xm010.dtb zynq-zc770-xm011.dtb \
+				zynq-zc770-xm012.dtb zynq-zc770-xm013.dtb \
+				zynq-ep107.dtb
-- 
1.7.0.4

